src/jarallax.esm.js

```diff
@@ -64,9 +65,29 @@ domReady(() => {
 // list with all jarallax instances
 // need to render all in one scroll/resize event
 const jarallaxList = [];
+let globalParallaxUpdateInit = false;
```

Comment:
I'd consider adding a comment to explain what this check is doing (along with commenting the new condition added which uses this). As I'm a bit confused what this bool is for.

Reply:
Comment added.